actors.md

```diff
@@ -1560,6 +1571,9 @@ func Approve(txid UInt) {
 	tx.Approved.Append(msg.From)
 
 	if len(tx.Approved) >= self.Required {
+		if !self.canSpend(tx.Value) {
+			Fatal("transaction amount exceeds available")
+		}
 		tx.RetCode = Send(tx.To, tx.Value, tx.Method, tx.Params)
```

Comment:
@whyrusleeping shall we change this to vm.Send to remain consistent with 1528?

Reply:
Yeah, thats probably the right thing to do.

Reply:
https://github.com/filecoin-project/specs/pull/416/commits/1428d0f1e46a0f78376ff30d78adec279465ceef